tests/formulagrader/test_formulagrader.py

```diff
@@ -215,8 +215,9 @@ def test_fg_userfunction():
         )
         grader(None, "that'sbad(1)")
 
-    expect = ("1 is not a valid key, must be of <type 'str'> for dictionary "
-              "value @ data\['user_functions'\]. Got {1: <ufunc 'tan'>}")
+    expect = ("1 is not a valid key, must be of {str_type} for dictionary "
+              "value @ data\['user_functions'\]. Got {{1: <ufunc 'tan'>}}").format(
+              str_type=str(str))
```

Comment:
Did you mean `type(str)` here?

Reply:
I'm changing this to `str_type=str`, see below.